www/README.md

```diff
@@ -36,25 +39,52 @@ For development you can serve the files locally by installing a webserver as fol
 sudo npm install -g http-server
 ```
 
-The server can then be launched:
+The server can then be launched from the `app` directory as follows:
 
 ```
-cd app
+cd www/app
 http-server -a localhost -p 8000
 ```
 
+### Building the app for production
+To build the application for production, run this command from the `www/master` directory:
+
+```
+npm run build
+```
+
+Like `npm start`, it runs `bower install` to install and/or update the framework dependencies, but then it runs `gulp build` to generate a production version of the application. The `build` target builds the application for production (e.g., with uglification of js files and minification of css files), and does not run a file watcher, so that it can be used in automated build environments.
+
+To make the production code available to the Kubernetes api server, run this command from the top level directory:
+
+```hack/build-ui.sh```
```

Comment:
Markdown formatting is broken for the rest of the document. There needs to be a line break on both ends of the ticks.